sdk/digitaltwins/Azure.DigitalTwins.Core/src/Queries/QueryBuilders/DigitalTwinsFunctions.cs

```diff
@@ -0,0 +1,131 @@
+        public static bool IsObject(object value) => throw new NotImplementedException();
+
+        /// <summary>
+        /// Adds the <see href="https://docs.microsoft.com/azure/digital-twins/reference-query-functions#is_null">IS_NULL</see> function to the condition statement of the query.
+        /// </summary>
+        /// <param name="value">The property that the query is looking for as null.</param>
+        /// <returns>True if the specified property is null within the scope of the query, false if otherwise.</returns>
+        public static bool IsNull(object value) => throw new NotImplementedException();
+
+        /// <summary>
+        /// Adds the <see href="https://docs.microsoft.com/azure/digital-twins/reference-query-functions#is_of_model">IS_OF_MODEL</see> function to the condition statement of the query.
+        /// </summary>
+        /// <param name="model">Model Id to check for.</param>
+        /// <param name="exact">Whether or not an exact match is required.</param>
+        /// <returns>True if the twin is of the specified model, false if otherwise.</returns>
+        public static bool IsOfModel(string model, bool exact) => throw new NotImplementedException();
+
+        /// <summary>
+        /// Adds the <see href="https://docs.microsoft.com/azure/digital-twins/reference-query-functions#is_of_model">IS_OF_MODEL</see> function to the condition statement of the query.
+        /// </summary>
+        /// <param name="model">Model Id to check for.</param>
+        /// <returns>True if the twin is of the specified model or a model inheriting from it, false if otherwise.</returns>
+        public static bool IsOfModel(string model) => throw new NotImplementedException();
+
+        /// <summary>
+        /// Adds the <see href="https://docs.microsoft.com/azure/digital-twins/reference-query-functions#startswith">STARTSWITH</see> function to the condition statement of the query.
+        /// </summary>
+        /// <param name="field">String to check the beginning of.</param>
+        /// <param name="prefix">String representing the beginning to check for.</param>
+        /// <returns>True if the prefix is found within the specified field, false if otherwise.</returns>
+        public static bool StartsWith(string field, string prefix) => throw new NotImplementedException();
+
+        /// <summary>
+        /// Adds the <see href="https://docs.microsoft.com/azure/digital-twins/reference-query-functions#endswith">ENDSWITH</see> function to the condition statement of the query.
+        /// </summary>
+        /// <param name="field">String to check the beginning of.</param>
+        /// <param name="suffix">String representing the beginning to check for.</param>
+        /// <returns>True if the suffix is found within the specified field, false if otherwise.</returns>
+        public static bool EndsWith(string field, string suffix) => throw new NotImplementedException();
```

Comment:
is this supposed to throw a not implemented exception? are you planning to implement it?

Reply:
The same question applies to all other functions.